File: SWIFT/Cap04/c04ex08.swift
// c04ex08.swift

import Foundation

var Transp: Character

print("Escolha o tipo de transporte:\n")
print("[M] - Motocicleta")
print("[S] - Scooter\n")

print("==> ", terminator: "")
if let input = readLine(), let t = input.uppercased().first {
    Transp = t
} else {
    fatalError("Entrada invalida.")
}

if (Transp == "M" || Transp == "S") {
    print("\nTransporte valido.")
} else {
    print("\nTransporte invalido.")
}

print("\nTecle <Enter> para encerrar... ", terminator: "")
_ = readLine()

File: PYTHON/Cap04/c04ex08.py
# c04ex08.py

print("Escolha o tipo de transporte:\n")
print("[M] - Motocicleta")
print("[S] - Scooter")
print()

transp = input("==> ").upper()

print()

if (transp == "M") or (transp == "S"):
  print("Transporte valido.")
else:
  print("Transporte invalido.")

input("\nTecle <Enter> para encerrar... ")
